fish_completions.go

```diff
@@ -28,9 +28,9 @@ function __%[1]s_debug
 end
 
 function __%[1]s_perform_completion
-    __%[1]s_debug "Starting __%[1]s_perform_completion with: $argv"
+    __%[1]s_debug "Starting __%[1]s_perform_completion"
 
-    set args (string split -- " " "$argv")
+    set args (string split -- " " (commandline -c))
```

Comment:
Given that you are already touching this line can you change it to
```suggestion
    set args (string split -- " " (string trim (commandline -c)))
```
Please check #1303 for details. I already checked that both `$argv` or `commandline -c` have to be trimmed

Reply:
Thanks @Luap99 !
I will update with your findings.

Reply:
Actually it has to be:
```suggestion
    set args (string split -- " " (string trim -l (commandline -c)))
```
Only trim the left spaces as the right ones are required if complete an empty arg (e.g. ` command [TAB]`).

Reply:
Done.
Thanks again!

Reply:
This line should probably be `set args (commandline -opc)`.
If the token at the cursor is really required, `(commandline -ct)` can be appended.